packages/wonder-blocks-data/src/util/response-cache.js

```diff
@@ -28,17 +30,17 @@ export class ResponseCache {
         return _default;
     }
 
-    _hydrationCache: MemoryCache;
-    _ssrOnlyCache: ?MemoryCache;
+    _hydrationCache: InMemoryCache;
+    _ssrOnlyCache: ?InMemoryCache;
```

Comment:
Now that we have a scoped in-memory cache, we could feasibly use one instance and two different scopes, but I think keeping them separate makes things a little easier to understand when each cache comes into play. It also makes it less likely that we introduce a bug like returning the ssr-only cache when we're asked to clone data for the hydration cache.